Simpsons_rule/simp.c

```diff
@@ -0,0 +1,44 @@
+
+// Girlscript Summer Of Code
+// Simpson's rule  in c
+//this is used to find integration 
```

Comment:
Please remove Girlscript comment and instead give description of problem statement.